Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -825,8 +825,10 @@ class DropDownMenu
     return item;
   }
 
-  private static OnItemClick (this: HTMLAnchorElement): boolean
+  private static OnItemClick (this: HTMLAnchorElement, ev: HTMLElementEventMap["click"]): boolean
```

Comment:
@a-ctor already suggested a better option for the type in the other PR.

Reply:
```suggestion
  private static OnItemClick (this: HTMLAnchorElement, ev: MouseEvent): boolean
```